implementations/swift/ockam/ockam_app/Ockam/AcceptingInvitation.swift

```diff
@@ -0,0 +1,237 @@
+                            accept_invitation(invitationIdContainer.id)
+                            self.closeWindow()
+                        },
+                        label: {
+                            Text("Accept")
+                        }
+                    )
+                    if state.enrolled {
+                        Button(
+                            action: {
+                                ignore_invitation(invitationIdContainer.id)
+                                self.closeWindow()
+                            },
+                            label: {
+                                Text("Ignore")
+                            }
+                        )
+                    }
+                    Button(
+                        action: {
+                            self.closeWindow()
+                        },
+                        label: {
+                            Text("Dismiss")
+                        }
+                    )
+                    .keyboardShortcut(.defaultAction)
+                }
+                .padding(8)
+                .background(.black.opacity(0.1))
+            } else if let (group, service) = self.service {
+                HStack {
+                    Spacer()
+                    ProfilePicture(url: group.imageUrl, size: 64)
+                    VStack(alignment: .leading) {
+                        if let name = group.name {
+                            Text(verbatim: name)
+                        }
+                        Text(verbatim: group.email)
+                    }
+                    Spacer()
+                }
+                .padding(25)
+
+                Text("The invitation is already accepted")
+                    .padding(5)
+                    .font(.headline)
+
+                Group {
+                    Text(service.sourceName)
+                    if let scheme = service.scheme {
+                        Text(verbatim: scheme).font(.caption)
+                    }
+                }.padding(0)
+
+                Spacer()
+                    .frame(height: 10)
+                Spacer()
+
+                HStack {
+                    Spacer()
+                    Button(
+                        action: {
+                            self.closeWindow()
+                        },
+                        label: {
+                            Text("Dismiss")
+                        }
+                    )
+                    .keyboardShortcut(.defaultAction)
+                }
+                .padding(8)
+                .background(.black.opacity(0.1))
+            } else {
+                Spacer()
+                Text("Invitation not found")
+                    .padding(.top, 8)
+                    .padding(.bottom, 0)
+                    .font(.headline)
+                Text("The invitation is either expired, was revoked or intended for a different account.\nPlease contact the other party for more information.")
```

Comment:
This invitation has either expired, was revoked, or was intended for a different account.\nPlease contact the sender of the invitation for more information.